includes/admin/class-wcmypa-admin.php

```diff
@@ -815,18 +810,26 @@ public function getProductOptions(): array
             ],
             'Country-of-origin' => [
                 'id'          => self::META_COUNTRY_OF_ORIGIN,
-                'label'       => __('Country of origin', 'woocommerce-myparcel'),
-                'type'        => 'text',
-                'description' => wc_help_tip(__('Country of origin is required for world shipments. Defaults to shop base.', 'woocommerce-myparcel')),
+                'label'       => __('country_of_origin', 'woocommerce-myparcel'),
+                'type'        => 'select',
+                'options'     => array_merge(
+                    [
+                      null => __("Default", "woocommerce-myparcel"),
+                    ],
+                    (new WC_Countries())->get_countries()
+                ),
+                'description' => wc_help_tip(
+                    __('setting_country_of_origin_help_text', 'woocommerce-myparcel')
+                ),
             ],
             'Age-check'         => [
                 'id'          => self::META_AGE_CHECK,
                 'label'       => __('shipment_options_age_check', 'woocommerce-myparcel'),
                 'type'        => 'select',
                 'options'     => [
-                    'Default',
-                    'Enabled',
-                    'Disabled',
+                    null                           => __("Default", "woocommerce-myparcel"),
```

Comment:
klopt het dat dit bij age check is aangepast?

Reply:
Ja de age check gaf een indexed array door, maar die werd in de method altijd overschreven door de age-check lijst met de juiste opties (null, yes en no), dat werkte omdat er geen andere select lists waren, maar toen ik de countries toevoegde kregen die dus ook altijd alleen maar de opties null, yes en no. vandaar dat ik de daadwerkelijke opties heb verplaatst naar de array

Reply:
wow :) nice